src/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp

```diff
@@ -296,94 +300,158 @@ class boss_shade_of_aran : public CreatureScript
                     }
                 }
                 NormalCastTimer = 1000;
-            } else NormalCastTimer -= diff;
+            }
+            else NormalCastTimer -= diff;
```

Comment:
I will sort this script out later, I was just getting good karazhan to some working progress so it is complete able.